Automattic/woocommerce-payments
# Fix Gravity Forms Product Add-ons on product WooPay Express Checkout Button

#### Changes proposed in this Pull Request








Fix Gravity Forms Product Add-ons fields data not send when clicking WooPay Express Checkout Button on product page.

#### Testing instructions





* Install Gravity Forms and Gravity Forms Product Add-ons plugins.
* Create a Gravity Form with Pricing Fields, set these fields to something the user can fill out valid and invalid data.
* Create a product that uses this Gravity Form.
* On the product page, fill out the data.
* Click the WooPay express checkout button.
* The data on WooPay should be loaded as expected.
* On the product page, fill out the data with some invalid data.
* Click the WooPay express checkout button.
* An error message with validation messages should be shown.

-------------------


**Post merge**
